source/MaterialXTest/ShaderValid.cpp

```diff
@@ -621,6 +623,7 @@ TEST_CASE("MaterialX documents", "[shadervalid]")
 #ifdef MATERIALX_BUILD_GEN_OSL
     mx::OslValidatorPtr oslValidator = createOSLValidator(orthographicView, oslLog);
     mx::ArnoldShaderGeneratorPtr oslShaderGenerator = std::static_pointer_cast<mx::ArnoldShaderGenerator>(mx::ArnoldShaderGenerator::create());
+    oslShaderGenerator->setRemappedShaderOutput(false);
```

Comment:
Thanks for adding this.

Reply:
For @niklasharrysson, for unit tests we don't want to try and add more workarounds for the Arnold of not handling structs on output so we turn of the final output remap for this particular test. It's still on by default otherwise.

Reply:
Ok, that's fine for now. I think for longer term we should move the output remapping to the ArnoldShaderGenerator, since it's only Arnold that has this problem. And then also make OslShaderGenerator a concrete class, which could be used for any situation when "vanilla" OSL is to be generated and tested.
But we can save this for another change list.